src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -4,7 +4,7 @@
 import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
-    private static final SessionFactory sessionFactory = initSessionFactory();
+    private static SessionFactory sessionFactory = initSessionFactory();
```

Comment:
Consider using a more robust initialization approach, such as lazy initialization with double-checked locking, to ensure thread safety and avoid unnecessary initialization if the session factory is not used.